src/PasswordReset.ts

```diff
@@ -98,7 +144,7 @@ export default class PasswordReset {
                 threepid_creds: creds,
                 threepidCreds: creds,
             }, this.password, this.logoutDevices);
-        } catch (err) {
+        } catch (err: any) {
```

Comment:
Proper type please

Reply:
Marking this as resolved to continue discussion here https://github.com/matrix-org/matrix-react-sdk/pull/9581#discussion_r1025756523